internal/metrics/handlers/options.go

```diff
@@ -2,29 +2,47 @@ package handlers
 
 import (
 	prometheusv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
-	prometheusalpha1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1alpha1"
+	cooprometheusv1alpha1 "github.com/rhobs/obo-prometheus-operator/pkg/apis/monitoring/v1alpha1"
 	mconfig "github.com/stolostron/multicluster-observability-addon/internal/metrics/config"
 	corev1 "k8s.io/api/core/v1"
 )
 
 type Options struct {
-	Platform      Collector
-	UserWorkloads Collector
-	Secrets       []*corev1.Secret
-	ClusterName   string
-	ClusterID     string
-	Images        mconfig.ImageOverrides
-	UI            UIOptions
+	Platform                 Collector
+	UserWorkloads            Collector
+	Secrets                  []*corev1.Secret
+	ClusterName              string
+	ClusterID                string
+	ClusterVendor            string
+	Images                   mconfig.ImageOverrides
+	IsHub                    bool
+	COOIsSubscribed          bool
+	CRDEstablishedAnnotation string
```

Comment:
I think I don't understand the reasoning behind this option. Can you please elaborate? It's passed to the `PrometheusOperatorAnnotations` in https://github.com/stolostron/multicluster-observability-addon/pull/314/files#diff-3ae1a829d6e7d4678f2944fedbda6e55edd6948ad1d7ea43adad91ef6b3f893dR192 and these annotations are good for what please? Sorry for my ignorance.